### PR TITLE
CM-508: add payroll files tab

### PR DESCRIPTION
**Pull request by jdolkowski**
https://openimis.atlassian.net/browse/CM-508

It needs: https://github.com/openimis/openimis-be-payroll_py/pull/41
payroll_file

### CODE REVIEW

File: src/components/payroll/PayrollPaymentFilesSearcher.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  files: state.payroll.payrollFiles,
+  pageInfo: state.payroll.payrollFilesPageInfo,
+  totalCount: state.payroll.payrollFilesTotalCount,
+});
+
+const mapDispatchToProps = (dispatch) => bindActionCreators({
+  fetchPayrollPaymentFiles,
+}, dispatch);
+
+export default connect(mapStateToProps, mapDispatchToProps)(PayrollPaymentFilesSearcher);
```
Comment (olewandowski1) [inline src/components/payroll/PayrollPaymentFilesSearcher.js:111]: Is there any reason why you're using mapStateToProps/mapDispatchToProps? Shouldn't we use useSelector/useDispatch instead?

**Review by olewandowski1 — COMMENTED** [review]
Single question from my side + take a look on lint warnings. @jdolkowski

File: src/components/payroll/PayrollPaymentFilesSearcher.js (hunk 1 above)
Comment (sniedzielski) [reply src/components/payroll/PayrollPaymentFilesSearcher.js:111]: just the matter of copy-paste mechanism. Due to limited time let's keep as it is. @olewandowski1

Comment (jdolkowski) [reply src/components/payroll/PayrollPaymentFilesSearcher.js:111]: ok then i am keeping it as it is

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**